meshroom/ui/qml/main.qml

```diff
@@ -558,6 +558,38 @@ ApplicationWindow {
         onTriggered: graphEditor.pasteNodes()
     }
 
+    Action {
+        id: cutAction
+
+        property string tooltip: {
+            var s = "Copy selected node"
+            s += (_reconstruction && _reconstruction.selectedNodes.count > 1 ? "s (" : " (") + getSelectedNodesName()
+            s += ") to the clipboard and remove them from the graph"
+            return s
+        }
+        text: "Cut Node" + (_reconstruction && _reconstruction.selectedNodes.count > 1 ? "s " : " ")
+        enabled: _reconstruction ? _reconstruction.selectedNodes.count > 0 : false
+        onTriggered: {
+            graphEditor.copyNodes()
+            graphEditor.uigraph.removeNodes(graphEditor.uigraph.selectedNodes)
+        }
+
+        function getSelectedNodesName()
```

Comment:
Same function than for copy. You can move it outside of the Action and use the same function.